docs/proposals/EntGAN.md

```diff
@@ -0,0 +1,45 @@
+# EntGAN: A Distributed GAN Framework Based on Multi-task
+## Motivation
+
+Generative adversarial networks(GAN) has been widely used to solve the challenges of small samples and heterogeneous data. In recent years, distributed GAN has expanded and accelerated the training of GAN. The existing distributed GANs aim to train a certain class of discriminators to generate a single kind of fake data. However, in real life, there are applications that need to train many kinds of discriminators at the same time, such as image conversion and robot inspection.     
+
+Therefore, we propose an enhanced GAN framework to help sole **multi-task** and **memory usage** problem. We call the framework EntGAN.     
```

Comment:
The motivation here is a little bit too abstract. What is the targeting scenario and dataset?

Reply:
In the new version, dataset is added in the proposal. But the story is still not yet strong enough considering limited edge resources or budgets. See https://github.com/kubeedge/sedna/pull/337#discussion_r940166481